scripts/tools/fetch-pull-request

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+if [ $# -eq 0 ]
+    then
+        echo "Id of pull request not supplied"
+        exit 0
+fi
+
+remote=${2-upstream}
+echo "Setting remote value as $remote"
+
+#fetching pull request
+fetch="git fetch $remote pull/$1/head:review-$1"
+echo "Exectuting $fetch"
+eval $fetch
```

Comment:
Also you directly commands instead of taking them in variable and then using eval. For eg, 
```
git fetch $remote pull/$1/head:review-$1
```